pkg/plugin/api/v1alpha1/platform/api.proto

```diff
@@ -71,6 +71,9 @@ message PlanPluginInput {
     string last_successful_config_file_name = 4;
     // The configuration of plugin that handles the deployment.
     bytes plugin_config = 5;
+    // The workking directory to use in plugin.
+    // This directory contains the cloned deploy source with the directory name "deploy_source".
+    string working_dir = 6;
```

Comment:
@khanhtc1202 
I think the working dir passed to the plugin should contain the deploy source.
On the other hand, this working_dir is used with `os.MkdirTemp`, so we should not place the deploy source at the root of this directory.
So, I documented that the deploy source is placed within the directory named "deploy_source."
WDYT?

Reply:
I think that's fine 👍